libcnb/src/target.rs

```diff
@@ -0,0 +1,36 @@
+pub struct ContextTarget {
+    /// The name of the target operating system.
+    ///
+    /// The value should conform to [Go's `$GOOS`](https://golang.org/doc/install/source#environment), for example
+    /// `linux` or `windows`.
+    ///
+    /// CNB `lifecycle` sources this value from the build OCI image's [`os` property](https://github.com/opencontainers/image-spec/blob/main/config.md#properties).
+    pub os: String,
+    /// The name of the target CPU architecture.
+    ///
+    /// The value should conform to [Go's $GOARCH](https://golang.org/doc/install/source#environment), for example
+    /// `amd64` or `arm64`.
+    ///
+    /// CNB `lifecycle` sources this value from the build OCI image's [`architecture` property](https://github.com/opencontainers/image-spec/blob/main/config.md#properties).
+    /// ``
+    pub arch: String,
+    /// The variant of the specified CPU architecture.
+    ///
+    /// The value should conform to [OCI image spec platform variants](https://github.com/opencontainers/image-spec/blob/main/image-index.md#platform-variants), for example
+    /// `v7` or `v8`.
+    ///
+    /// CNB `lifecycle` sources this value from the build OCI image's [`variant` property](https://github.com/opencontainers/image-spec/blob/main/config.md#properties).
+    pub arch_variant: Option<String>,
+    /// The name of the operating system distribution. Should be empty for windows.
```

Comment:
```suggestion
    /// The name of the operating system distribution. Should be empty for Windows.
```

Reply:
Fixed in [c5cb54c](https://github.com/heroku/libcnb.rs/pull/773/commits/c5cb54ce9285983331e3afb08accbb45e970c542)